### PR TITLE
Speaker Ankita Shukla

### PR DESCRIPTION
**Pull request by ankitashukla**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -786,6 +786,9 @@ <h1 class="large-h1 text-white">The Internet of Things for Me<div><div>at Asia's
 								<span>Open Invention Network</span>
 							</div>
 						</div>
+						
+						
+
```
Comment (rctay) [inline index.html:812]: you're adding blank lines - am I missing something here?

**Comment by mariobehling** [comment]
@ankitashukla Please add your details to the index.html. Thanks.

**Comment by ankitashukla** [comment]
Can you please merge the PR? 

Thanks!

**Comment by rctay** [comment]
Closed by b9df8764977c6db77c39e0e9b8abfc286bc49701

**Pull request closed without merging**